### PR TITLE
Skip dotnet watch tests on arm

### PR DESCRIPTION
**Pull request by jkotalik**
https://github.com/dotnet/aspnetcore/issues/23360

### CODE REVIEW

File: src/Tools/dotnet-watch/test/GlobbingAppTests.cs [hunk 1]
```diff
@@ -21,9 +21,10 @@ public GlobbingAppTests(ITestOutputHelper logger)
             _app = new GlobbingApp(logger);
         }
 
-        [Theory]
+        [ConditionalTheory]
         [InlineData(true)]
         [InlineData(false)]
+        [SkipOnHelix("https://github.com/dotnet/aspnetcore/issues/23360", Queues = "Debian.9.Arm64;Debian.9.Arm64.Open")]
```
Comment (BrennanConroy) [inline src/Tools/dotnet-watch/test/GlobbingAppTests.cs:27]: Do we know if that name works? The name in msbuild is `(Debian.9.Arm64.Open)Ubuntu.1804.Armarch.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-9-helix-arm64v8-a12566d-20190807161036`

**Review by BrennanConroy — APPROVED** [review]
:'(

**Review by dougbu — APPROVED** [review]
Looks good but per @BrennanConroy's comment, please manually run the helix-matrix pipeline (a bit after office hours) to confirm the tests aren't run everywhere you intend to skip them

**Comment by jkotalik** [comment]
Worked; other test failures will be fixed by https://github.com/dotnet/aspnetcore/pull/23356.

**Pull request merged**